### ISSUE TITLE
"Expand/Collapse Tree" button of Data Explorer page is not accessible through keyboard.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1012]
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" Menu item and select it.
Navigate through all the controls under "Data Explorer" blade.
Navigate to "Expand/Collapse Tree" button using tab key and press enter key or spacebar to activate it.
Verify the issue.

Actual Result:
"Expand/Collapse Tree" button of Data Explorer page is not accessible through keyboard.

Expected Result:
"Expand/Collapse Tree" button of Data Explorer page which is accessible using mouse should also be accessible through keyboard.

https://user-images.githubusercontent.com/8519407/132722093-c4c75fb7-cea5-41c2-af4f-dc5d68f7c6fc.mp4

**Comment by ato9000** [comment]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216293
